themes-default/slim/views/config_postProcessing.mako

```diff
@@ -21,531 +25,94 @@ const startVue = () => {
+                header: 'Post Processing',
+                presets: [
+                    '%SN - %Sx%0E - %EN',
+                    '%S.N.S%0SE%0E.%E.N',
+                    '%Sx%0E - %EN',
+                    'S%0SE%0E - %EN',
+                    'Season %0S/%S.N.S%0SE%0E.%Q.N-%RG'
+                ],
+                processMethods: processMethods,
+                pattern: config.postProcessing.naming.pattern,
+                multiEpStrings: multiEpStrings,
+                multiEpSelected: config.postProcessing.naming.multiEp,
+
+                enabledSports: config.postProcessing.naming.enableCustomNamingSports,
+                sportsPattern: config.postProcessing.naming.patternSports,
+
+                enabledAirByDate: config.postProcessing.naming.enableCustomNamingAirByDate,
+                abdPattern: config.postProcessing.naming.patternAirByDate,
+                
+                enabledAnime: config.postProcessing.naming.enableCustomNamingAnime,
+                animePattern: config.postProcessing.naming.patternAnime,
+                animeMultiEpStrings: multiEpStrings,
+                animeMultiEpSelected: config.postProcessing.naming.animeMultiEp,      
+                animeNamingType: config.postProcessing.naming.animeNamingType,
+                seriesDownloadDir: config.postProcessing.seriesDownloadDir,
+                processAutomatically: config.postProcessing.processAutomatically,
+                processMethod: config.postProcessing.processMethod,
+                deleteRarContent: config.postProcessing.deleteRarContent,
+                unpack: config.postProcessing.unpack,
+                noDelete: config.postProcessing.noDelete,
+                reflinkAvailable: config.postProcessing.reflinkAvailable,
+                postponeIfSyncFiles: config.postProcessing.postponeIfSyncFiles,
+                autoPostprocessorFrequency: config.postProcessing.autoPostprocessorFrequency || 10,
+                airdateEpisodes: config.postProcessing.airdateEpisodes,
+                moveAssociatedFiles: config.postProcessing.moveAssociatedFiles,
+                allowedExtensions: config.postProcessing.allowedExtensions || [],
+                addShowsWithoutDir: config.postProcessing.addShowsWithoutDir,
+                createMissingShowDirs: config.postProcessing.createMissingShowDirs,
+                renameEpisodes: config.postProcessing.renameEpisodes,
+                postponeIfNoSubs: config.postProcessing.postponeIfNoSubs,
+                nfoRename: config.postProcessing.nfoRename,
+                syncFiles: config.postProcessing.syncFiles || [],
+                fileTimestampTimezone: config.postProcessing.fileTimestampTimezone || 'local',
+                timezoneOptions: timezoneOptions,
+                extraScripts: config.postProcessing.extraScripts || [],
+                extraScriptsUrl: config.postProcessing.extraScriptsUrl,
+                appNamingStripYear: config.postProcessing.appNamingStripYear
```

Comment:
Move these all to computed so if you include the config mapping you'd have something like this

```
computed: Object.assign(store.mapState(['config']), {
    extraScripts(){
        return this.config.postProcessing.extraScripts || [];
    }
}),
```

Reply:
Make sure to add all the fields to the `store.config`.